checker/manual/manual.bbl

```diff
@@ -45,14 +45,13 @@ Werner Dietl, Michael~D. Ernst, and Peter M{\"u}ller.
   Conference}, pages 333--357, Lancaster, UK, July~27--29, 2011.
 
 \bibitem[EJM{\etalchar{+}}14]{ErnstJMDPRKBBHVW2014}
-Michael~D. Ernst, Ren{\'e} Just, Suzanne Millstein, Werner Dietl, Stuart
+Michael~D. Ernst, Ren{\'e} Just, Suzanne Millstein, Werner~M. Dietl, Stuart
```

Comment:
Why the update to this file??
I don't have the "M." in papers.
You loose page numbers in a paper below and completely remove one paper.
Undo this file change.